tests/reconfiguration.py

```diff
@@ -303,6 +305,45 @@ def test_join_straddling_primary_replacement(network, args):
     return network
 
 
+def test_retiring_nodes_emit_at_most_one_signature(network, args):
+    primary, _ = network.find_primary()
+
+    # Force ledger flush of all transactions so far
+    network.get_latest_ledger_public_state()
+    ledger = ccf.ledger.Ledger(primary.remote.ledger_paths())
+
+    NODES = "public:ccf.gov.nodes.info"
+    SIGS = "public:ccf.internal.signatures"
```

Comment:
These are already defined as `ccf.ledger.NODES_TABLE_NAME` and `ccf.ledger.SIGNATURE_TX_TABLE_NAME`